migration/_layout_migration.adoc

```diff
@@ -56,7 +56,7 @@ endif::production_build[]
 
 // Post Migration
```

Comment:
Daniel, I don't know if you'd want to, but we could edit the comments throughout to match the headings themselves, removing spaces and hyphens following _pre_ and _post_ (to reduce cognitive dissonance for anyone who's paying attention to these things).